sdcm/nemesis.py

```diff
@@ -466,6 +467,27 @@ def disrupt_rolling_restart_cluster(self):
             raise UnsupportedNemesis('RollingRestartCluster is supported only for kubernetes backends')
         self.cluster.rollout_restart()
 
+    def disrupt_authenticator_switch_and_restart(self):
+        self._set_current_disruption('AuthenticatorSwitchAndRestart %s' % self.target_node)
+        if not self._is_it_on_kubernetes():
+            raise UnsupportedNemesis('AuthenticatorSwitchAndRestart is supported only for kubernetes backends')
+        # Switch to com.scylladb.auth.SaslauthdAuthenticator
+        if self.cluster.params.get('prepare_saslauthd'):
+            if 'PasswordAuthenticator' in self.cluster.params.get('authenticator'):
+                update_authenticator(self.cluster.nodes, 'com.scylladb.auth.SaslauthdAuthenticator')
```

Comment:
What happens to all connections that are already connected?
(like all other c-s that are already running)

Reply:
Nothing effects as expected. cqlsh login interface (parameters) is completely same after the change.

Reply:
ok.